### PR TITLE
sys-fs/mount-zip: Add mount-zip

### PR DESCRIPTION
**Pull request by fdegros**

### CODE REVIEW

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (Flowdalic) [inline sys-fs/mount-zip/mount-zip-1.0.9.ebuild:4]: EAPI 8?

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+inherit toolchain-funcs
+
+DESCRIPTION="FUSE file system for ZIP archives"
+HOMEPAGE="https://github.com/google/mount-zip"
+SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE=""
```
Comment (Flowdalic) [inline sys-fs/mount-zip/mount-zip-1.0.9.ebuild:15]: Drop empty variable definition.

**Review by Flowdalic — CHANGES_REQUESTED** [review]

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild (hunk 1 above)
Comment (fdegros) [reply sys-fs/mount-zip/mount-zip-1.0.9.ebuild]: Done

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild (hunk 2 above)
Comment (fdegros) [reply sys-fs/mount-zip/mount-zip-1.0.9.ebuild]: Done

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="FUSE file system for ZIP archives"
+HOMEPAGE="https://github.com/google/mount-zip"
+SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (Flowdalic) [inline sys-fs/mount-zip/mount-zip-1.0.9.ebuild:10]: The project appears to be licensed under GPL-3 or any later version, so this should probably be `GPL-3+`.

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+RDEPEND="
+	dev-libs/icu
```
Comment (Flowdalic) [inline sys-fs/mount-zip/mount-zip-1.0.9.ebuild:15]: Shouldn't this use the `=` slot-operator for icu and libzip? So, e.g., `dev-libs/icu:=`.

**Review by Flowdalic — CHANGES_REQUESTED** [review]

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+EAPI=8
+
+DESCRIPTION="FUSE file system for ZIP archives"
+HOMEPAGE="https://github.com/google/mount-zip"
+SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```
Comment (Flowdalic) [inline sys-fs/mount-zip/mount-zip-1.0.9.ebuild:12]: Have you tested the ebuild on both arm platforms?

I would suggest to remove the arm keywords and fill a keywording request for them instead.

**Review by Flowdalic — CHANGES_REQUESTED** [review]

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild (hunk 4 above)
Comment (fdegros) [reply sys-fs/mount-zip/mount-zip-1.0.9.ebuild]: Done

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild (hunk 3 above)
Comment (fdegros) [reply sys-fs/mount-zip/mount-zip-1.0.9.ebuild]: Done

File: sys-fs/mount-zip/mount-zip-1.0.9.ebuild (hunk 5 above)
Comment (fdegros) [reply sys-fs/mount-zip/mount-zip-1.0.9.ebuild]: > Have you tested the ebuild on both arm platforms?

Yes.

I work on ChromeOS, which uses the same `ebuild`-based build system as Gentoo. ChromeOS builds and uses `mount-zip` on all its supported architectures, including AMD-64, ARM-64 and ARM-32. I have personally tested `mount-zip` on all these architectures. Moreover, we run automated tests exercising `mount-zip` on all these architectures as well. Finally, ChromeOS has a comprehensive crash reporting system. I looked at the crash reports and the different usage metrics, and I am confident that `mount-zip` works correctly and reliably on all these architectures. And it has been working for several years.

**Comment by thesamesam** [comment]
Thank you!

**Pull request closed without merging**